plugins/lightspeed-backend/src/service/router.test.ts

```diff
@@ -1,5 +1,4 @@
-import { getVoidLogger } from '@backstage/backend-common';
-// import { mockCredentials, mockServices } from '@backstage/backend-test-utils';
+import { mockServices } from '@backstage/backend-test-utils';
 import { MockConfigApi } from '@backstage/test-utils';
```

Comment:
We should use mock services [here](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/lightspeed-backend/src/service/router.test.ts#L58) instead.